### PR TITLE
Upgrade docs to Play 2.5.0

### PR DESCRIPTION
**Pull request by marcospereira**

### CODE REVIEW

File: docs/manual/working/scalaGuide/main/sql/slick/code/slick.sbt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+// #add-library-dependencies
+libraryDependencies += "com.typesafe.play" %% "play-slick" % "2.0.0"
+// #add-library-dependencies
+
+// #add-dependency-with-evolutions
+libraryDependencies ++= Seq(
+  "com.typesafe.play" %% "play-slick" % "1.1.1"
+  "com.typesafe.play" %% "play-slick-evolutions" % "1.1.1"
```
Comment (dotta) [inline docs/manual/working/scalaGuide/main/sql/slick/code/slick.sbt]: The version should be updated to 2.0.0

**Comment by dotta** [comment]
Thanks @marcospereira! If you could rebase I'd be happy to merge this one.

**Comment by marcospereira** [comment]
Done. :-)

File: docs/manual/working/scalaGuide/main/sql/slick/code/slick.sbt (hunk 1 above)
Comment (marcospereira) [inline docs/manual/working/scalaGuide/main/sql/slick/code/slick.sbt]: Fixed. Thanks, @dotta.

**Comment by wsargent** [comment]
LGTM

**Pull request merged**